src/ImageLoaders/MzExe/PharLapExtender.cs

```diff
@@ -72,7 +72,10 @@ public override Program LoadProgram(Address? addrLoad)
             if ((fileHeader.flags & FlagImagePacked) != 0)
             {
                 UnpackImage(fileHeader, image);
+            } else {
```

Comment:
Consider adhering to the coding format in the code base: `{` opens a new line always.

Reply:
oh cool yes, I'll fix that and the whitespace as soon as I can tonight.

Thanks very much!

Reply:
fixed now! thanks again

-dc